src/BenchmarksDriver/Program.cs

```diff
@@ -87,6 +87,9 @@ public static int Main(string[] args)
                 "Source dependency. Format is 'repo@branchOrCommit'. " +
                 "Repo can be a full URL, or a short name under https://github.com/aspnet.",
                 CommandOptionType.MultipleValue);
+            var threadCountOption = app.Option("-t|--threadCount",
+                "Kestrel ThreadCount.",
+                CommandOptionType.SingleValue);
```

Comment:
Still think you should rename `--threads` to `--clientThreads`, but that can be addressed in a separate PR.